### PR TITLE
Support flexible horizontal expansion

### PR DESCRIPTION
**Pull request by XCXCXCXCX**
## What is the purpose of the change
#649 support a way to easier config cluster
To support flexible horizontal expansion

## Brief changelog

1. Added synchronization record in datum{KEY=com.alibaba.nacos.naming.serverList, value=com.alibaba.nacos.naming.cluster.servers.Servers@Object}
2. Added interface com.alibaba.nacos.naming.controllers.OperatorController#updateServerList
3. Allow the interface to be used to synchronize the cluster.conf configuration of each node in the nacos cluster to achieve horizontal expansion.

## Verifying this change

Three nodes are started in the idea, which are 127.0.0.1:8848, 127.0.0.1:8849,127.0.0.1:8850.
At this point, the cluster.conf configuration is: [127.0.0.1:8848,127.0.0.1:8849,127.0.0.1:8850]
Access any node via the /nacos/v1/ns/operator/servers interface and return a value of 200[OK].
At this point, the fourth node 127.0.0.1:8851 is started, and its cluster.conf configuration is [127.0.0.1:8848,127.0.0.1:8849,127.0.0.1:8850,127.0.0.1:8851].
Break the point, access other interfaces, process the request normally, and the peers list in RaftCore has been updated to 4 nodes {127.0.0.1:8848,127.0.0.1:8849,127.0.0.1:8850,127.0.0.1:8851}.

### CODE REVIEW

File: naming/src/main/java/com/alibaba/nacos/naming/controllers/OperatorController.java [hunk 1]
```diff
@@ -217,6 +218,17 @@ public JSONObject getHealthyServerList(HttpServletRequest request) {
...
     }
 
+    /**
+     * update the static file cluster.conf
+     * @param clusterHosts
+     * @return clusterHosts in the static file cluster.conf
+     */
+    @RequestMapping(value = "/servers", method = RequestMethod.PUT)
+    public String updateServerList(@RequestParam("clusterHosts") List<String> clusterHosts) throws Exception {
+        serverListManager.updateServers(clusterHosts);
```
Comment (nkorange) [inline naming/src/main/java/com/alibaba/nacos/naming/controllers/OperatorController.java]: 应该有个能增量修改的参数，只需要传入新增的或者减少的节点信息

**Comment by chuntaojun** [comment]
The node management sink has moved to the core module since version 1.3.0, so pr needs to be redesigned

**Comment by chuntaojun** [comment]
You can implement a RaftGroup group managed by a cluster node using LogProcessor4CP and MemberLookup of the 1.3.0 conformance protocol layer

**Comment by KomachiSion** [comment]
> The node management sink has moved to the core module since version 1.3.0, so pr needs to be redesigned

> You can implement a RaftGroup group managed by a cluster node using LogProcessor4CP and MemberLookup of the 1.3.0 conformance protocol layer

PR close

**Pull request closed without merging**